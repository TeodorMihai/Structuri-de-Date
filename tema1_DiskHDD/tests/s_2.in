2 10
::w 0 0 ABCD
1
::w 0 1 DDDD
2
::r 0 0 
15
::r 0 1
10
::d 0 0
100
::w 1 15 AABB
13
::d 1 15
5
::d 1 15
5
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
1
::w 1 2 AAAA
1
::w 9 7500 AAAA
10000
::d 1 2
30000
::d 9 7500
3
::w 6 7 ACCA
5000
::r 5 4
9
::r 6 7
2
::w 3 1 0001
245
::d 6 7
10000
::r 7 8
2
::r 7 1
2
::r 7 10
3
::e
