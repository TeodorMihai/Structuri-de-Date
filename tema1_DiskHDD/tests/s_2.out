DDDD
33
ABCD
1
35
34
33
ACCA
0000
3.85 3.28 3.24 3.26
